virttest/guest_agent.py

```diff
@@ -877,6 +877,46 @@ def guest_exec_status(self, pid):
         cmd = "guest-exec-status"
         return self._cmd_args_update(cmd, pid=pid)
 
+    def ssh_add_authorized_keys(self, username, *keys, reset=None):
```

Comment:
No docstring for `reset`, and recommend to define `keys` at the end.

Reply:
added docstring for reset, but in the presence of multiple positional parameters, it seems that positional parameters cannot be placed last, right? I tried that, but case was failed and something like I said could find in log.

Reply:
@6-dehan You are right, 'positional argument follows keyword argument'.